website/server/models/user/schema.js

```diff
@@ -521,6 +521,7 @@ let schema = new Schema({
     return {};
   }},
   pushDevices: [PushDeviceSchema],
+  ABtest: {type: String},
```

Comment:
Rule of thumb:

* If the client should not be able to see it, mark it as private
* If the client should be able to see it but third party integrations should not use it, mark it as internally private by giving it a leading `_`
* If the client should be able to see and integrations should use it, leave it as is